app/bundles/CampaignBundle/Form/Type/CampaignLeadSourceType.php

```diff
@@ -27,14 +27,18 @@ class CampaignLeadSourceType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $sourceType = $options['data']['sourceType'];
+        $sourceChoices = $options['source_choices'] ?? null;
+        foreach($sourceChoices as $key => $val) {
+            $sourceChoices[$key] = $val.' #'.$key;
+        }
```

Comment:
I would use parentheses to display the ID as I think that's already done in some other select box.
```suggestion
        foreach($sourceChoices as $key => $val) {
            $sourceChoices[$key] = $val.' ('.$key.')';
        }
```